src/main/java/com/shrralis/ssdemo1/controller/MapRestController.java

```diff
@@ -12,56 +12,52 @@
 
 import com.shrralis.ssdemo1.dto.MapDataDTO;
 import com.shrralis.ssdemo1.entity.MapMarker;
+import com.shrralis.ssdemo1.service.interfaces.IIssueService;
 import com.shrralis.ssdemo1.service.interfaces.IMapMarkersService;
 import com.shrralis.tools.model.JsonResponse;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.security.access.annotation.Secured;
 import org.springframework.web.bind.annotation.*;
 
 import java.time.LocalDateTime;
 
-@CrossOrigin
 @RestController
-@RequestMapping("map")
+@RequestMapping(value = "/map", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Is it possible to set json globally so we will not specify it on every controller ?